src/Apps/Artist/ArtistApp.tsx

```diff
@@ -28,13 +23,7 @@ const ArtistApp: React.FC<ArtistAppProps> = ({ artist, children }) => {
     <>
       <ArtistMetaFragmentContainer artist={artist} />
 
-      <AnalyticsContext.Provider
-        value={{
-          contextPageOwnerId: artist.internalID,
-          contextPageOwnerSlug,
-          contextPageOwnerType,
-        }}
-      >
+      <AnalyticsContextProvider contextPageOwnerId={artist.internalID}>
```

Comment:
I see, it requires some arguments. Maybe set these with a hook vs reinitializing the context? Or would that get too unwieldy 🤔

Reply:
You only need to provide the ID if it's a /:slug route. I could also enforce/warn on that 🤔